index.js

```diff
@@ -0,0 +1,257 @@
+  document.querySelector('#button-two'),
+  document.querySelector('#button-three'),
+  document.querySelector('#button-four'),
+];
+// through all button items
+buttonArray.forEach((buttonItem, index) => {
+  const contentArray = [
+    content1,
+    content2,
+    content3,
+    content4,
+  ];
+  const item = contentArray[index];
+  buttonItem.addEventListener('click', () => {
+    const popupCard = document.createElement('div');
+    sectionCard.appendChild(popupCard);
+    // inner html for pop-up
+    popupCard.innerHTML = `
+      <!--popup mobile-->
+      <div class="target-mobile">
+        <div id="popup-card-menu" class="popup-card" data-target="p-1">
+            <div class="card-popup">
+                <div class="popup-header">
+                    <h2>${item.title}</h2>
+                    <img id="closing" class="closePopup" src="images/IconB.png" alt="closing">
+                </div>
+                <div class="canopy">
+                    <h4>CANOPY</h4>
+                    <ul class="back">
+                        <li><img class="dot" src="images/Counter.png" alt="dot">Back End Dev</li>
+                        <li><img class="dot" src="images/Counter.png" alt="dot">2015</li>
+                    </ul>
+                </div>
+                <div class="popup-body">
+                    <img src="${item.image}" alt="Snapshoot-Portfolio">
+                    <div class="popup-body-desktop">
+                    <div class="p-div-desktop">
+                        <p>
+                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been
+                            the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley
+                            of type and scrambled it to make a type specimen book.
+                        </p>
+                    </div>
+                    <div class="side">
+                        <ul class="small-btn-div">
+                            <li class="small-btn"><a>html</a></li>
+                            <li class="small-btn"><a>css</a></li>
+                            <li class="small-btn"><a>javaScript</a></li>
+                        </ul>
+                        <ul class="small-btn-div">
+                            <li class="small-btn"><a>github</a></li>
+                            <li class="small-btn"><a>ruby</a></li>
+                            <li class="small-btn"><a>Bootstrap</a></li>
+                        </ul>
+                        <hr>
+                        <div class="popup-button">
+                            <button class="icon2"><a href="${item.buttonLive}">See live</a></button>
+                            <button class="icon3"><a href="${item.buttonSource}">See Source</a></button>
+                        </div>
+                    </div>
+                </div>
+                </div>
+            </div>
+        </div>
+      </div>
```

Comment:
- You have done nicely so far,  but there are some differences between your popup and what is in the design template. Please check the marked sections in the images attached below. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your Project:
- In the figma template, the width of the image is almost the entire width of the poup modal. Please fix this 🙏 
- Also, please remove the `text-underline` in your live links. Please fix this 🙏 
- Remember to change the text of the live-links when the buttons are hovered on for a better User interface please fix this 👍 
<img width="795" alt="mmdtyjsry" src="https://user-images.githubusercontent.com/73607512/217771646-54ad5c56-ce07-47ba-bc75-ba09427943c5.PNG">

Figma template:
<img width="514" alt="dgadaerga" src="https://user-images.githubusercontent.com/73607512/217771731-b36f3b5c-252a-482f-9615-77a2aa8e7246.PNG">